src/components/verticaltabs/_verticaltabs.scss

```diff
@@ -69,6 +70,46 @@
   position: absolute;
   right: 20px;
 }
+.cmp-verticaltabs__tablist {
+  counter-reset: item; 
+}
+.cmp-verticaltabs__tablist>li::before{
```

Comment:
Please create a common mixin for both vertical and horizontal tab

Reply:
Created!